src/components/common/TranscriptMetadataCard.tsx

```diff
@@ -108,7 +110,7 @@ const TranscriptMetadataComponent = ({
               <div className="flex flex-wrap gap-2">
                 {(topics && topics.length > 0) ?
                   topics.map((topic) => (
-                    <Pill key={topic} name={topic} slug={`/tags/${createSlug(topic)}`} />
+                    <Pill key={topic.slug} name={topic.name || ""} slug={`/tags/${topic.slug}`} />
```

Comment:
topic.name will always be defined. you are using the slug as the fallback in contentlayer